src/plugin/memory-webview-main.ts

```diff
@@ -55,11 +55,14 @@ const columnConfigurations = [
     manifest.CONFIG_SHOW_VARIABLES_COLUMN,
 ];
 
-export class MemoryWebview {
+export class MemoryWebview implements vscode.CustomEditorProvider {
     public static ViewType = `${manifest.PACKAGE_NAME}.memory`;
     public static ShowCommandType = `${manifest.PACKAGE_NAME}.show`;
     public static VariableCommandType = `${manifest.PACKAGE_NAME}.show-variable`;
 
+    private readonly _onDidChangeCustomDocument = new vscode.EventEmitter<vscode.CustomDocumentEditEvent>();
+    public readonly onDidChangeCustomDocument = this._onDidChangeCustomDocument.event;
```

Comment:
From some of the comments on the VSCode issue, it sounds like we could implement this functionality by adding functionality in our current debug session watchers to listen for [memory events](https://microsoft.github.io/debug-adapter-protocol/specification#Events_Memory). Would we like that to be part of the eventual implementation?

Reply:
That could make sense. I'm trying to get to a point where we can offer an alternative memory viewer for testing purposes and restart the conversation on that issue. I fully expect the architecture to change beyond a basic setting, but that needs to be driven by the vscode team IMO